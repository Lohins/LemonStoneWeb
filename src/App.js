import React from 'react';
import './App.css';

const App = () => (
  <div className="App">
    <h1> Hello, World Sitong! </h1>
  </div>
);

export default App;
