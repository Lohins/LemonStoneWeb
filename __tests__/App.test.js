import renderer from 'react-test-renderer';
import React from 'react';
import App from '~src/App';

test('Test app rendering', ()=> {
	const wrapper = renderer.create(<App />).toJSON();
	expect(wrapper).toMatchSnapshot();
});
