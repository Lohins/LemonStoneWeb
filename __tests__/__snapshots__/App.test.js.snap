// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test app rendering 1`] = `
<div
  className="App"
>
  <h1>
     Hello, World Sitong! 
  </h1>
</div>
`;
